server/actions/votes.go

```diff
@@ -199,6 +199,10 @@ func CountVotes(c *gin.Context) {
 		c.String(http.StatusBadRequest, "Can't count votes of unfinalized election")
 		return
 	}
+	if len(election.Votes) == 0 {
+		c.String(http.StatusBadRequest, "Election has no votes")
+		return
+	}
```

Comment:
Is this really a bad request?
Feels like it should be something else. Anyway, I think we could leave it as is right now since it does not matter.


https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400

Reply:
I suppose 500 would be more appropriate.